app/presenters/deploy_group_usage_csv_presenter.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require 'csv'
+
+class DeployGroupUsageCsvPresenter
+  def self.to_csv
+    CSV.generate do |csv|
+      csv << csv_header
+      Project.all.each do |project|
+        if project.stages.empty?
+          csv << csv_line(project)
+        else
+          project.stages.each do |stage|
+            stage.deploy_groups.each do |deploy_group|
+              csv << csv_line(project, stage, deploy_group)
```

Comment:
same logic here when a stage has no deploy group ?

Reply:
if `DEPLOY_GROUP_FEATURE` is enabled - Can you have a `stage` without a `deploy_group`? The UI seemed to enforce that relationship...

Reply:
ah true!